### PR TITLE
Changes for Keras auto histogram logging

### PR DESCRIPTION
**Pull request by dsblank**
This PR:

* sets three parameters in Experiment for each histogram type
* adds questions to ask user
* changes the default to yes/no for optimizer to match the others
* requires later version of keract

### CODE REVIEW

File: python/cookiecutter.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "online_or_offline": ["Online", "Offline"],
   "framework": ["keras"],
   "confusion_matrix": ["Yes", "No"],
-  "histogram": ["Yes", "No"],
+  "weight_histograms": ["Yes", "No"],
+  "gradient_histograms": ["Yes", "No"],
+  "activation_histograms": ["Yes", "No"],
   "embedding": ["Yes", "No"],
-  "optimizer": ["No", "Yes"]
+  "optimizer": ["Yes", "No"]
```
Comment (Lothiraldan) [inline python/cookiecutter.json:11]: The first value is the default one if the user don't want to choose and most users probably won't want to start with the Optimizer and Optimizer won't work in a server without internet connection so I think it's the right default value.

**Review by Lothiraldan — COMMENTED** [review]
LGTM but let's wait for the next SDK to be released first, we can also bump the minimum version of the comet_ml package to ensure that users use a compatible version.

File: python/cookiecutter.json (hunk 1 above)
Comment (dsblank) [reply python/cookiecutter.json:11]: I found it confusing for anything other than the default case where you take all of the defaults. If you actually read them, and the yes's and no's keep changing, then it is confusing. If there were a way to make one be the default without changing the order, then I agree.

Comment (dsblank) [reply python/cookiecutter.json:11]: Let's try it like this to see which is better.

**Pull request merged**